### ISSUE TITLE
Users cant login after password reset

### ISSUE THREAD

**Issue by cacpmw** [global-devteam/laravel-api-password-reset#24]
> When I made the DELETE with my token, my password was changed and the password_resets record was cleared. However, my password was not changed to what I passed in. I suspect that this is because the password is [hashed using bcrypt](https://github.com/global-devteam/laravel-api-password-reset/blob/f05a4f26ae6c6600b1ee7afd108b5211aa7e5922/src/app/Services/PasswordResetService.php#L76) rather than Hash::make as the default Laravel uses.
> 3. If the configuration for bcrypt is set to false, the [password is not hashed at all](https://github.com/global-devteam/laravel-api-password-reset/blob/f05a4f26ae6c6600b1ee7afd108b5211aa7e5922/src/app/Services/PasswordResetService.php#L78-L79) and stored plaintext. I think that still the user won’t be able to login.

**Comment by cacpmw** [comment]
@beauraines this didnt happen when I reset the password using the package

**Comment by beauraines** [comment]
With this configuration file, my password is reset to the plain text value and thus, I cannot login.


```php
<?php

return [
    'route' => [
        'prefix'     => 'api/v1/auth/',
        'uri'        => 'password',
        'middleware' => 'api',
    ],
    'tokenSize'             => 60,
    'emailNotificationLink' => 'some/link/',
    'bcryptPassword'        => true,

];
```




```
> select email,password from users where email = 'beau.raines@gmail.com'

+ ---------- + ------------- +
| email      | password      |
+ ---------- + ------------- +
| beau.raines@gmail.com | $2y$10$M98srLg9zJYd7f8zmyXMu.qOW9/gKDjpEF/oLxnHbDHtKL9UXoka2 |
+ ---------- + ------------- +
1 rows

```

DELETE /api/v1/auth/password
```json
{
	"email":"beau.raines@gmail.com",
	"password":"secret",
	"password_confirmation":"secret",
	"token": "jErCb8ZgyhmE6jprMEw8WiNxjnOXmPBgNQcxm67fxC67NTIIq7B1neXKKX5F"
}
```

I receive the successful email

and then

```sql
> select email,password from users where email = 'beau.raines@gmail.com'

+ ---------- + ------------- +
| email      | password      |
+ ---------- + ------------- +
| beau.raines@gmail.com | secret        |
+ ---------- + ------------- +
1 rows

```

Maybe we can get together Monday to work on this. I will have time between 14:30 - 15:30 Brazil time to screenshare.

**Comment by cacpmw** [comment]
@beauraines were you working with the latest updates that I've made? I think  this was a problem with bcryptPassword configuration that I might've solved already.

As of now we can get together on Monday at that time. I might have to postpone it tho. I will let you know. Sending calendar invite now

**Comment by beauraines** [comment]
I'm pretty sure I pulled the latest master. I've accepted your meeting for Monday! Talk to you then.

**Comment by beauraines** [comment]
Closing. As you suggested, I reinstalled from master (and resolved #35 )and it works.
